File: Julia/fizzbuzz.jl
# naive implementation
for i = 1:typemax(UInt64)
    str = ""
    if i % 3 == 0
        str = str * "fizz"
    end
    if i % 5 == 0
        str = str * "buzz"
    end
    if str == ""
        str = string(i)
    end
    println(str)
end

File: Rust/fizzbuzz.rs
/*
 * naive implementation
 */

fn main() {

    for i in 0..u64::MAX {
        let mut s = String::new();
        if i % 3 == 0 {
            s += "fizz";
        }
        if i % 5 == 0 {
            s += "buzz";
        }
        if s == "" {
            s += &format!("{}", i)[..];
        }
        println!("{}", s);
    }
}
